components/twitter/twitter.app.mjs

```diff
@@ -1076,6 +1076,26 @@ export default {
         config,
       }));
     },
+    async unfollowUser(opts = {}) {
```

Comment:
You can get the method parameters unstructured here

```suggestion
    async unfollowUser({params, $}) {
```

Reply:
@Jordon-L @lcaresia seems to have two extra spaces before `params` and `$`